app/controllers/case_logs_controller.rb

```diff
@@ -107,7 +107,7 @@ def responses_for_page(page)
 
       if question_info["type"] == "checkbox"
         question_info["answer_options"].keys.reject { |x| x.match(/divider/) }.each do |option|
-          result[option] = question_params.include?(option)
+          result[option] = question_params.include?(option) ? "Yes" : "No"
```

Comment:
This looks a little weird. It won't work for actual boolean fields (which will save "No" as `true`, and probably not for integer fields either?) are checkbox questions consistently stored as strings? Looks like some are ints and some booleans from the schema?

Reply:
Ah, just noticed the boolean checkboxes, they should really be integer. I'll fix that